### PR TITLE
Traduction de Monolithic-apps.md

### LINKED ISSUE #1267: [FR] Localize monolithic-apps for French
Localization of monolithic-apps.md into french.

### PR DESCRIPTION
**Pull request by huats**
### Describe your changes
This PR localizes monolithic-apps concept to French

### Related issue number or link (ex: `resolves #issue-number`)
Resolves #1267

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/monolithic-apps.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: Applications Monolithiques
+status: Completed
+category: concept
+tags: ["architecture", "", ""]
+---
+
+## Ce que c'est
+
+Une application monolithique contient toutes les fonctionnalités au sein d'un unique logiciel déployable.
+C'est souvent la manière la plus simple pour commencer lors développement d'une application.
```
Comment (Krast76) [inline content/fr/monolithic-apps.md:11]: pour commencer "le" développement d'une application

File: content/fr/monolithic-apps.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Cependant, lorsque l'application se complexifie, monolithes peuvent devenir difficiles à maintenir.
```
Comment (Krast76) [inline content/fr/monolithic-apps.md:12]: "les monolithes" peuvent devenir difficiles à...

File: content/fr/monolithic-apps.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Avec plus de développeurs qui travaillent sur la même base de code, la probabilité de conflits dans les changements et donc de communication directe entre les développeurs augmente.
+
+## Problème auquel il répond
+
+Transformer une applications en [microservices](/microservices/) la surcharge opérationelle - comme il y a plus de choses à tester, déployer et à maintenir opérationel.
```
Comment (Krast76) [inline content/fr/monolithic-apps.md:17]: J'aurais plutôt dit : 
"Transformer une application en microservices augmente la charge de travail."

En l'état la phrase doit être reformulé je pense.

File: content/fr/monolithic-apps.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Au début du cycle de développement d'un produitm il peut être avantageux de repousser cette complexité et de construire une application monolithique jusqu'à ce que le produit soit reconnu comme un succès.
```
Comment (Krast76) [inline content/fr/monolithic-apps.md:18]: "d'un produitm" -> "d'un produit"

File: content/fr/monolithic-apps.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+## Quelle en est l'utilité
+
+Un monolith correctement architecturé peut respecter les principes lean en étant la manière la plus simple d'obtenir une application fonctionnelle.
+Lorsque la valeur commerciale d'une application monolithique s'avère un succès, elle peut être cécomposée en microservices.
```
Comment (Krast76) [inline content/fr/monolithic-apps.md:23]: "s'avère être un succès,"

File: content/fr/monolithic-apps.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+Créer une application microservices avant que la valeur soit attestée peut savérer une dépense prénaturée d'ingénieurie.
```
Comment (Krast76) [inline content/fr/monolithic-apps.md:24]: peut s’avérer être une dépense prématuré.

File: content/fr/monolithic-apps.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+Si l'application n'amène aucune valeur, cet effort sera gaché.
```
Comment (Krast76) [inline content/fr/monolithic-apps.md:25]: gâché.

**Review by fydrah — APPROVED** [review]

**Review by Krast76 — APPROVED** [review]

**Pull request merged**